mining/src/lib.rs

```diff
@@ -153,3 +160,20 @@ impl core::ops::Sub for &MempoolCountersSnapshot {
         }
     }
 }
+
+/// Contains a snapshot of only the RPC transaction counter and time elapsed
+pub struct RpcTxCounterSnapshot {
```

Comment:
`RpcTxCounterSnapshot ` -> `P2pTxCounterSnapshot `

(low priority are P2P txs)

Reply:
Maybe call it `P2pTxCountSample`

Reply:
Addressed in [4894f8f](https://github.com/kaspanet/rusty-kaspa/pull/389/commits/4894f8fb2837e950884c829e645029a9d9299869)